Sources/SwiftSyntax/Documentation.docc/Macro Versioning.md

```diff
@@ -0,0 +1,29 @@
+# Updating a Macro to a New Major Swift-syntax Version
```

Comment:
Nitpick: `Swift-syntax` here vs `swift-syntax` below. Personally I wouldn't title case it

Reply:
Damn, I didn’t double-check the output of titlecase.com here